deploy/catalog/Chart.yaml

```diff
@@ -1,3 +1,3 @@
-name: catalog
-description: Kubernetes incubator Service Catalog deployment Helm chart.
+name: kubernetes-service-catalog
```

Comment:
I think this should be disambiguated here because this should probably be globally unique.  @arschles @krancour is that supposed to be the case with helm charts?